.github/workflows/build.yml

```diff
@@ -14,63 +14,37 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       fail-fast: false
-      matrix:
-        python-version: ["3.9"]
     timeout-minutes: 50
 
     steps:
-      - name: Checkout main
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-
-      - name: Checkout lndocs
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           repository: laminlabs/lndocs
           ssh-key: ${{ secrets.READ_LNDOCS }}
           path: lndocs
           ref: main
-
       - uses: actions/setup-python@v4
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "3.10"
           cache: "pip"
-          cache-dependency-path: ".github/workflows/build.yml" # See dependencies below
-
-      - name: Install Nextflow
-        uses: nf-core/setup-nextflow@v1
+          cache-dependency-path: ".github/workflows/build.yml"
+      - uses: nf-core/setup-nextflow@v1
         with:
           version: "latest-stable"
-
       - name: Cache Nextflow output
```

Comment:
I wasn't able to figure out whether this is actually used or not. It seems like the nextflow pipeline is actually running, doesn't it?

I do think that an automatic caching by-passing the run would make a lot of sense to better iterate on this use case. 

The CI times are waaaaaaaaay tooooooooo loooooong to iterate fast.

(I wasn't keen on installing nextflow on my laptop...)

Reply:
Yeah I couldn't get it to work. I'm pretty sure that this is the output folder and I expected the `nextflow resume` command to pick it up, but for some reason it doesn't.

Reply:
Then let's just remove this from the build.yml?

Reply:
Did so while resolving the conflict!